roles/core/time/templates/chrony.conf.j2

```diff
@@ -3,22 +3,18 @@
 ## {{ansible_managed}}
 
 # External servers/pool if asked
-{% if external_time.time_server.server and time_profile == 'server' %}
+{% if time_profile == 'server' %}
 {% for external_server in external_time.time_server.server %}
 server {{external_server}}
 {% endfor %}
-{% endif %}
-{% if external_time.time_server.pool and time_profile == 'server' %}
 {% for external_pool in external_time.time_server.pool %}
 pool {{external_pool}} iburst maxsources 3
 {% endfor %}
 {% endif %}
-{% if external_time.time_client.server and time_profile == 'client' %}
+{% if time_profile == 'client' %}
```

Comment:
There is no default value for `external_time.time_client.server`. Anyway, see comment above and revert these changes please.